android/app/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,10 +1,8 @@
-• OpenStreetMap login works again!
-• Better isolines elevation data for European countries
-• OpenStreetMap data as of March 26
-• Bookmark lists are sorted by modification time (recent at the top)
-• Eye icons for visible/hidden bookmark lists
-• Fixed crash when importing GPX, KML, KMZ from the Files app
-• Bigger "View search results on the map" button
-• FreeDrive mode and faster bookmarks in Android Auto
+• Upgraded to new OpenStreetMap authentication method - all previously authenticated users must re-login!
+• OpenStreetMap data as of April 29
+• Display difficult hiking trails as brown dotted, expert ones as black dotted
+• Dark blue lines for dedicated cycleways
+• Blue lines under white/brown dashes for shared foot/cycle paths
+• Roads visible under semi-transparent bridge outlines
```

Comment:
This is a minor change, better to add keywords-friendly and more important one:

- Fixed GPX import from Google Files app